ruby-common.macros

```diff
@@ -21,3 +21,13 @@
 %rubydevelSTOP() %nil
 %rubydevelxSTOP() %*
 #
+
+%fix_sheb() \
```

Comment:
1. the fix_sheb is ugly ... spell it out.
2. ruby_fix_shebang or rb_fix_shebang is better.
3. I miss error handling in case rb_suffix is not defined.